### PR TITLE
Edge case fix for user who has rated all possible recommendation cocktails

### PR DESCRIPTION
**Pull request by mkrumholz**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Rubocop has passed locally and any fixes were made for failures
- [ ] CircleCI build is passing
- [x] All tests passing locally

## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: 
closes #

## What is the new behavior?

- 
- 
- 

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Review by zachjamesgreen — APPROVED** [review]

File: app/services/recommendation_service.rb [hunk 1]
```diff
@@ -19,9 +19,10 @@ def self.recommendation(user_id)
     weighted_baselines = baselines.groupby('cocktail_id').sum()[[user_id,'weightedRating']]
     ratings_complete = weighted_baselines_with_counts(baselines, weighted_baselines)
     unrated = remove_rated(cocktail_ratings, ratings_complete, user_id)
+    return nil if unrated.empty
```
Comment (markcyen) [inline app/services/recommendation_service.rb:22]: really great find on edge case

File: app/services/recommendation_service.rb [hunk 2]
```diff
@@ -19,9 +19,10 @@ def self.recommendation(user_id)
...
     recommendations = Pandas.DataFrame.new()
     weighted_avg(recommendations, unrated)
-    top_recommendation(recommendations).reset_index['cocktail_id'].to_i
+    top_recommendation(recommendations).reset_index['cocktail_id'].to_i unless recommendations.empty
```
Comment (markcyen) [inline app/services/recommendation_service.rb:25]: another one...

**Review by markcyen — APPROVED** [review]
Sweet.

**Review by twarbelow — COMMENTED** [review]
look into uniqueness in the future

**Pull request merged**